staging/src/k8s.io/apiserver/pkg/apis/audit/v1/types.go

```diff
@@ -219,88 +243,104 @@ type PolicyRule struct {
 	//  "/healthz*" - Log all health checks
 	// +optional
 	NonResourceURLs []string `json:"nonResourceURLs,omitempty" protobuf:"bytes,7,rep,name=nonResourceURLs"`
 
-	// OmitStages is a list of stages for which no events are created. Note that this can also
+	// A list of stages for which no events are created. Note that this can also
 	// be specified policy wide in which case the union of both are omitted.
 	// An empty list means no restrictions will apply.
+	// Each item in the list can be one of:
+	// - 'RequestReceived': The stage for events generated as soon as the audit handler receives
+	//   the request, and before it is delegated down the handler chain.
+	// - 'ResponseStarted': The stage for events generated once the response headers are sent,
+	//   but before the response body is sent. This stage is only generated for long-running requests
+	//   (e.g. watch).
+	// - 'ResponseComplete': The stage for events generated once the response body has been completed,
+	//   and no more bytes will be sent.
+	// - 'Panic': The stage for events generated when a panic occurred.
 	// +optional
 	OmitStages []Stage `json:"omitStages,omitempty" protobuf:"bytes,8,rep,name=omitStages"`
 
-	// OmitManagedFields indicates whether to omit the managed fields of the request
+	// This  indicates whether to omit the managed fields of the request
 	// and response bodies from being written to the API audit log.
 	// - a value of 'true' will drop the managed fields from the API audit log
 	// - a value of 'false' indicates that the managed fileds should be included
 	//   in the API audit log
 	// Note that the value, if specified, in this rule will override the global default
 	// If a value is not specified then the global default specified in
-	// Policy.OmitManagedFields will stand.
+	// `policy.omitManagedFields` will stand.
 	// +optional
 	OmitManagedFields *bool `json:"omitManagedFields,omitempty" protobuf:"varint,9,opt,name=omitManagedFields"`
 }
 
 // GroupResources represents resource kinds in an API group.
 type GroupResources struct {
-	// Group is the name of the API group that contains the resources.
-	// The empty string represents the core API group.
+	// The name of the API group that contains the resources.
+	// An empty string represents the core API group.
 	// +optional
 	Group string `json:"group,omitempty" protobuf:"bytes,1,opt,name=group"`
+
 	// Resources is a list of resources this rule applies to.
 	//
 	// For example:
-	// 'pods' matches pods.
-	// 'pods/log' matches the log subresource of pods.
-	// '*' matches all resources and their subresources.
-	// 'pods/*' matches all subresources of pods.
-	// '*/scale' matches all scale subresources.
+	// - `pods` matches pods.
+	// - `pods/log` matches the log subresource of pods.
+	// - `*` matches all resources and their subresources.
+	// - `pods/*` matches all subresources of pods.
+	// - `*/scale` matches all scale subresources.
```

Comment:
I wonder if we should single quotes here. We don't have a strong convention of being able to treat code comments as valid Markdown and we should be wary of adopting that by stealth.

Reply:
@sftim You may want to read the OpenAPI spec if you don't have an idea about the assumption of having markdown style description for types and fields. The fact is that strings like these are making the API docs confusing, and the confusion is not limited to the configuration APIs but also all API references. I can give you a lot of other examples if needed.